public/docs/_examples/toh-6/ts/sample.css

```diff
@@ -1,7 +0,0 @@
-/* #docregion */
-.error {color:red;}
```

Comment:
Notice the `.error` style here. It seems to have been lost in your proposed changes.